docs/architecture-decisions/0000-template.md

```diff
@@ -0,0 +1,22 @@
+# ADR 0000: [title]
+
+<!-- These documents have names that are short noun phrases. For example, "ADR 1: Deployment on Ruby on Rails 3.0.10" or "ADR 9: LDAP for Multitenant Integration" -->
```

Comment:
Update the above two parts to reflect that we call them eg ADR001 with three digits